src/docfx/lib/HtmlUtility.cs

```diff
@@ -195,15 +195,15 @@ public static (List<(string uid, int line, int column)>, string) TransformXrefs(
                 var logWarning = node.GetAttributeValue("data-throw-if-not-resolved", null);
                 var rawSource = node.GetAttributeValue("data-raw-source", null);
                 var rawHtml = node.GetAttributeValue("data-raw-html", null);
-                var raw = HttpUtility.HtmlDecode(!string.IsNullOrEmpty(rawHtml) ? rawHtml : rawSource) ?? xref.Value;
+                var raw = HttpUtility.HtmlDecode(!string.IsNullOrEmpty(rawHtml) ? rawHtml : rawSource) ?? $"@{xref.Value}";
```

Comment:
We can use innerHtmas last fallback as in v2 `$"@{xref.Value}"` => `node.InnerHtml`

Reply:
simply the fallback for now only takes `raw`. fix it later if we need to